ddtrace/context.py

```diff
@@ -10,7 +11,11 @@
 from .constants import USER_ID_KEY
 from .internal.compat import NumericType
 from .internal.compat import PY2
+from .internal.constants import _TRACEPARENT_KEY
+from .internal.constants import _TRACESTATE_KEY
 from .internal.logger import get_logger
+from .internal.utils.http import w3c_format_known_propagated_tags
+from .internal.utils.http import w3c_format_unknown_propagated_tags
```

Comment:
we should avoid leaking this function in the public api. Sorry I should've noticed this earlier:

```suggestion
from .internal.utils.http import w3c_format_known_propagated_tags as _w3c_format_known_propagated_tags
from .internal.utils.http import w3c_format_unknown_propagated_tags as _w3c_format_unknown_propagated_tags
```